lib/discourse_slack/slack.rb

```diff
@@ -120,10 +120,20 @@ def self.get_key(id = nil)
     def self.set_filter_by_id(id, channel, filter, tags = nil, channel_id = nil)
       data = get_store(id)
```

Comment:
or maybe not - perhaps that'd be suboptimal when finding which channels need to be notified.

That said, I think there are still some issues regarding how tag subscriptions work.  I'll create a separate topic on meta to discuss.